### ISSUE TITLE
Venge swap doesn't cancel CM ult with shard

### ISSUE THREAD

**Issue by bobswinkle** [ValveSoftware/Dota2-Gameplay#1901]
### Ability name

Nether Swap & Freezing field

### Description

When CM has shard and casts Freezing Field, Vengeful Spirit Nether Swap no longer cancels her channelling. The tooltip for Vengeful Spirit says doing so will cancel channelling and the shard says it can still be interrupted. 

### Example Match ID (and possibly Timestamp)

6673825969

**Comment by shaileshgarg** [comment]
Venge Swap gives a Stop Command to Swapped Units, which cancels the Chanelling, but CM ulti with Shard is not channeled anymore, so a stop command doesn't cancel it.

**Comment by bobswinkle** [comment]
> Venge Swap gives a Stop Command to Swapped Units, which cancels the Chanelling, but CM ulti with Shard is not channeled anymore, so a stop command doesn't cancel it.

If this is an intended interaction then the Freezing Field tooltip needs updating. It should say "stuns can still interrupt"

**Comment by deanrih** [comment]
> > Venge Swap gives a Stop Command to Swapped Units, which cancels the Chanelling, but CM ulti with Shard is not channeled anymore, so a stop command doesn't cancel it.
> 
> If this is an intended interaction then the Freezing Field tooltip needs updating. It should say "stuns can still interrupt"

Well it is still technically correct, Nether Swap **does not stun** the target, just give a **stop command**, something like Magic Missile does stun so it should cancel/stop Freezing Field's spell.

**Comment by deanrih** [comment]
> > Venge Swap gives a Stop Command to Swapped Units, which cancels the Chanelling, but CM ulti with Shard is not channeled anymore, so a stop command doesn't cancel it.
> 
> If this is an intended interaction then the Freezing Field tooltip needs updating. It should say "stuns can still interrupt"

Yes, it is intended and yes, the bug is on the shard's tooltip rather than the mechanic itself, also I believe it can be interrupted by either Stun and Silence, I'm not sure if Root and Leash interrupt it though. But either way, the shard description will be very long if it is gets "fixed".

**Comment by TheUnholyWrath** [comment]
#111

**Comment by shaileshgarg** [comment]
This can be closed.
